### PR TITLE
db-query header.php & syntax

### PR DESCRIPTION
**Pull request by Udera**
Update of db-query in header.php

### CODE REVIEW

File: vexim/config/header.php [hunk 1]
```diff
@@ -13,9 +15,9 @@
     print     "-- " . $_SESSION['domain'] . " ";
   }
   if (isset($_SESSION['domain_id'])) {
-    if (($domheaderrow['enabled'] == "0") || ($domheaderrow['enabled'] == "f")) {
+    if (($domheaderrow['enabled'] === "0") || ($domheaderrow['enabled'] === "f")) {
```
Comment (rimas-kudelis) [inline vexim/config/header.php]: I don't think "f" is actually a valid value here. Integer 0, on the other hand, seems very valid. We should check what is actually returned by the query.

Same for below.

Comment (Udera) [inline vexim/config/header.php]: This "f" was inserted here:
https://github.com/vexim/vexim2/blob/1e807f14b2bd1d9d9cc9b918983e03bcb600c2f6/vexim/config/header.php

Do you see why this was put there?

Comment (Udera) [inline vexim/config/header.php]: Was this put for a new feature, or debugging? The `enabled`-field is either `1` or `0`

**Comment by Udera** [comment]
Ok, I removed this condition completely. I haven't seen where the variable could be 'f', we will see in the final testing, if this causes an error.

**Comment by rimas-kudelis** [comment]
Have you tested if your patch works? In particular, I'm interestėd whether `$domheaderrow['enabled']` and `$usrheaderrow['enabled']` are indeed of string type, and not integers.

**Comment by Udera** [comment]
I checked through the list and used the pull requests #82 #84.
And if you put `if (($domheaderrow['enabled'] === 0))` instead, it won't work, so mysql seems to return everything as string.

**Comment by rimas-kudelis** [comment]
Cool!

**Pull request merged**